packages/components/atoms/f-form-field/src/components/FormField.vue

```diff
@@ -45,8 +61,9 @@
                 placeholder=" "
                 :data-test-id="testId.input"
                 :class="[
-                    $style['c-formField-input'],
-                    (isSelectionControl ? $style['c-formField-input--focus'] : '')
+                    $style['c-formField-field'],
+                    $style['c-formField-field--defaultHeight'],
+                    (isSelectionControl ? $style['c-formField-field--focus'] : '')
```

Comment:
Could use the Vue conditional syntax here? i.e. `{ [$style['c-formField-field--focus']]: isSelectionControl }`.